server/app/config.js

```diff
@@ -53,7 +51,7 @@ app.use(
 // Uncomment one or more of these options depending on the format of the data sent by your client:
 
 app.use(express.json());
-app.use(express.urlencoded());
+// app.use(express.urlencoded());
```

Comment:
you no longer need express.urlencoded() ?

Reply:
no this is an error ! :)

Reply:
ah no ! actually I commented again this line because I thought I needed it but finally no ! :)